test/ui/segmented-bar/segmented-bar-test.js

```diff
@@ -0,0 +1,32 @@
+var TestController = require("montage-testing/test-controller").TestController;
+
+exports.SegmentedBarTest = TestController.specialize({
+    content: {value: [
+        {"region": "region1", "size": 10, "flag": true},
+        {"region": "region2", "size": 10, "flag": true},
+        {"region": "region3", "size": 10, "flag": true},
+        {"region": "region4", "size": 10, "flag": true},
+        {"region": "region5", "size": 10, "flag": true},
+        {"region": "region6", "size": 50, "flag": true}
+    ]},
+    tree: {value: {
+        region: "USA",
+        size: 99,
+        regions: [
+            {
+                region: "Canada",
+                size: 20
+            },
+            {
+                region: "Ivory Coast",
+                size: 30,
+                regions: [
+                    {
+                        region: "China",
+                        size: 40
+                    }
+                ]
+            }
+        ]
+    }}
+});
```

Comment:
nit: That JSON formatting looks strange: Why is value on the same line as its parent on both places it appears? For that matter, why is there no blank line between the property descriptors, unlike every other `specialize()` call in this pull request?